### ISSUE TITLE
Kong plugin CI environment

### ISSUE THREAD

**Issue by peter-evans** [Kong/kong-plugin-prometheus#29]
Would be great if you could open source the `kong-ci` repository that is being used here. That repository plus the travis config could then become the standard way to test third-party plugins. Perhaps there could be an advanced version of the [kong-plugin](https://github.com/Kong/kong-plugin) template that includes this travis build setup.

This `kong-ci` repository:
https://github.com/Kong/kong-plugin-prometheus/blob/master/.travis.yml#L49

**Comment by Tieske** [comment]
Yes, @peter-evans that would certainly be nice. We're actually already having an internal discussion about it, exactly in the way you described.

ping @gszr
